create-project.mdx

```diff
@@ -9,21 +9,21 @@ create a project through the Hypermode Console or Hyp CLI.
 ## Create with Hypermode Console
 
 <Note>
-  Hypermode relies on git as a source for project deployment. Through the
+  Hypermode relies on Git as a source for project deployment. Through the
   project creation flow, you'll connect Hypermode to your GitHub account.
 </Note>
 
 From your organization home, click **New Project**. Set a name for the project
 and click **Create**.
 
-Once you have created a project, Hypermode prompts you to import the repo with
-your Modus app. To create your first Modus app, see the
-[Modus quickstart](modus/quickstart).
+Once you have created a project, Hypermode prompts you to finish setting up your
```

Comment:
I wouldn't assume the user doesn't already have a Modus app here

Reply:
Agreed, I'll add that back in